core/src/dird/dird_conf.cc

```diff
@@ -124,7 +124,7 @@ static ResourceItem dir_items[] = {
   { "DirSourceAddress", CFG_TYPE_ADDRESSES_ADDRESS, ITEM(res_dir,  DIRsrc_addr), 0, CFG_ITEM_DEFAULT, "0", NULL, NULL },
   { "QueryFile", CFG_TYPE_DIR, ITEM(res_dir, query_file), 0, CFG_ITEM_REQUIRED, NULL, NULL, NULL },
   { "WorkingDirectory", CFG_TYPE_DIR, ITEM(res_dir, working_directory), 0, CFG_ITEM_DEFAULT | CFG_ITEM_PLATFORM_SPECIFIC, PATH_BAREOS_WORKINGDIR, NULL, NULL },
-  { "PidDirectory", CFG_TYPE_DIR, ITEM(res_dir, pid_directory), 0, CFG_ITEM_DEFAULT | CFG_ITEM_PLATFORM_SPECIFIC, PATH_BAREOS_PIDDIR, NULL, NULL },
+  { "PidDirectory", CFG_TYPE_DIR, ITEM(res_dir, pid_directory), 0, CFG_ITEM_DEPRECATED | CFG_ITEM_DEFAULT | CFG_ITEM_PLATFORM_SPECIFIC, "unused", NULL, NULL },
```

Comment:
```suggestion
  { "PidDirectory", CFG_TYPE_DIR, ITEM(res_dir, pid_directory), 0, CFG_ITEM_DEPRECATED | CFG_ITEM_DEFAULT | CFG_ITEM_PLATFORM_SPECIFIC, "unused", "-21.0.0", "This directive is deprecated and unused." },
```

Reply:
Is "unused" really a good default value or should we put an empty string here?

Reply:
1. I don't know, I did not like an empty string either. Maybe we want no default at all?
2. there is manually added documentation files, that have been updated to explain the situation. Thus we probably don't need the in-line docs here.
3. what effect does the "-21.0.0" have?